core-plugins/src/main/java/co/cask/hydrator/plugin/transform/PythonEvaluator.java

```diff
@@ -344,6 +347,12 @@ private Object decodeUnion(Object object, List<Schema> schemas) {
   }
 
   private void init() {
+
+    // This is necessary to prevent 'ImportError' being thrown by Jython
+    Properties properties = new Properties();
+    properties.put("python.import.site", "false");
```

Comment:
since you are setting this to false, i am wondering if they were available before, am not sure if they were.